WBooks/BookRentals/BookRentalsCard/BookRentalsCardViewController.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  BookRentalsCardViewController.swift
+//  WBooks
+//
+//  Created by sebastian.a.spadea on 26/04/2022.
+//
+
+import UIKit
+import WolmoCore
+
+final class BookRentalsCardViewController: UIViewController {
+    private let cellIdentifier = "LibraryViewCell"
+    private let bookRentalsCardViewModel: BookRentalsCardViewProtocol
+    private let bookRentalsCardView = BookRentalsCardView()
+
+    init(bookRentalsCardViewModel: BookRentalsCardViewModel) {
+        self.bookRentalsCardViewModel = bookRentalsCardViewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func loadView() {
+        view = bookRentalsCardView
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        guard let table = bookRentalsCardView.rentalsTableView else { return }
+        table.register(
+            UINib(nibName: cellIdentifier, bundle: nil),
+            forCellReuseIdentifier: cellIdentifier)
+        table.backgroundColor = .backgroundGray
+        table.delegate = self
+        table.dataSource = self
```

Comment:
I think this part of the code can go in a method that calls something like setupRentalTable